windows-observ-lib/panels.libsonnet

```diff
@@ -0,0 +1,405 @@
+        commonlib.panels.all.timeSeries.base.new(
+          'Context switches/Interrupts',
+          targets=[
+            t.systemContextSwitches,
+            t.systemInterrupts,
+          ],
+          description=|||
+            Context switches occur when the operating system switches from running one process to another. Interrupts are signals sent to the CPU by external devices to request its attention.
+
+            A high number of context switches or interrupts can indicate that the system is overloaded or that there are problems with specific devices or processes.
+          |||
+        ),
+      systemExceptions:
+        commonlib.panels.all.timeSeries.base.new(
+          'System calls and exceptions',
+          targets=[
+            t.windowsSystemExceptions,
+            t.windowsSystemCalls,
+          ],
+        ),
+      systemThreads:
+        commonlib.panels.all.timeSeries.base.new(
+          'System threads',
+          targets=[
+            t.windowsSystemThreads,
+          ],
+        ),
+      timeNtpStatus:
+        commonlib.panels.system.statusHistory.ntp.new(
+          'NTP status',
+          targets=[t.timeNtpStatus],
+          description=''
+        )
+        + g.panel.timeSeries.standardOptions.withNoValue('No data. Please check that "time" collector is enabled.'),
+      timeNtpDelay:
+        commonlib.panels.all.timeSeries.base.new(
+          'NTP delay',
+          targets=[
+            t.timeNtpDelay,
+            t.timeOffset,
+          ],
+          description=|||
+            NTP trip delay: Total roundtrip delay experienced by the NTP client in receiving a response from the server for the most recent request,
+            in seconds. This is the time elapsed on the NTP client between transmitting a request to the NTP server and receiving a valid response from the server.
+
+            Time offset: Absolute time offset between the system clock and the chosen time source, in seconds.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('seconds')
+        + g.panel.timeSeries.standardOptions.withNoValue('No data. Please check that "time" collector is enabled.'),
+      cpuCount: commonlib.panels.cpu.stat.count.new(targets=[t.cpuCount]),
+      cpuUsageTs: commonlib.panels.cpu.timeSeries.utilization.new(targets=[t.cpuUsage]),
+      cpuUsageTopk: commonlib.panels.all.timeSeries.topkPercentage.new(
+        title='CPU usage',
+        target=t.cpuUsage,
+        topk=25,
+        instanceLabels=this.config.instanceLabels,
+        drillDownDashboardUid=this.dashboards.overview.uid,
+      ),
+      cpuUsageStat: commonlib.panels.cpu.stat.usage.new(targets=[t.cpuUsage]),
+      cpuUsageByMode: commonlib.panels.cpu.timeSeries.utilizationByMode.new(
+        targets=[t.cpuUsageByMode],
+        description=|||
+          CPU usage by different modes.
+        |||
+      ),
+      // TODO add why it is important, consider alert
```

Comment:
Seems like the description adequately explains why this metric is important?
```suggestion
```

Reply:
thnks, removed